guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -23,10 +23,10 @@ endif::[]
 
 |/opt/puppetlabs | 500 MB | Not Applicable
 
-ifdef::katello,satellite,orcharhino[]
+ifdef::satellite,orcharhino[]
```

Comment:
```suggestion
ifdef::katello,satellite,orcharhino[]
```

Maybe the wording in https://github.com/theforeman/foreman-documentation/pull/4085/files#r2242381801 was off, but what I wanted to say: You don't have to exclude orcharhino from Katello & Satellite on line 29 for now -> So line 26-29 for Katello-like builds is good.

Reply:
Well, the IoP procedures are currently enabled only for Satellite. So no containers storage requirement in Katello and orcharhiho.

Reply:
But it affects line 27 with the Pulp storage dir.